exercises/practice/bottle-song/spec/bottle_song_spec.cr

```diff
@@ -0,0 +1,131 @@
+require "spec"
+require "../src/*"
+
+describe "BottleSong" do
+  describe "verse" do
```

Comment:
I'm happy to change it if you'd rather have it be flat. The canonical data actually has 3 layers and I omitted the third because it seemed like a bit much.

Reply:
I think it is fine as it is, but 3 is too much, so good call.